Dockerfile

```diff
@@ -64,7 +64,7 @@ COPY --from=busybox-min /etc/ssl/cert.pem /etc/ssl/cert.pem
 
 # Install relayer user
 COPY --from=busybox-min /etc/passwd /etc/passwd
-COPY --from=busybox-min --chown=1025:1025 /home/relayer /home/relayer
+COPY --from=busybox-min --chown=100:1000 /home/relayer /home/relayer
```

Comment:
Looks like you wanted the UID of 100

Reply:
yep! 100:1000 is what it was in the previous rly images